CHANGELOG.md

```diff
@@ -31,7 +31,9 @@ Latest
 * Removed compatibility with standalone ``wcsaxes`` and instead depend on the
   version in astropy 1.3. SunPy now therefore depends on astropy>=1.3.
 * Update to `TimeRange.__repr__`; now includes the qualified name and `id` of
-  the object.
+the object.
```

Comment:
the indentation on the wrap line is important and you should also mention the change in API for `ImageAnimator` (update_plot).